storm-core/src/clj/backtype/storm/ui/core.clj

```diff
@@ -293,10 +293,17 @@
               (bolt-comp-summs id))]
     (sort-by #(-> ^ExecutorSummary % .get_executor_info .get_task_start) ret)))
 
-(defn worker-log-link [host port topology-id]
+(defn worker-log-link [host port topology-id secure?]
   (let [fname (logs-filename topology-id port)]
-    (url-format (str "http://%s:%s/log?file=%s")
-          host (*STORM-CONF* LOGVIEWER-PORT) fname)))
+    (if secure?
```

Comment:
Actually I just had a thought.  We probably want this to be 
```
(if (and secure? (*STORM-CONF* LOGVIEWER-PORT))
...)
```

That way someone is not required to secure their logviewers if they secure the main UI.